packages/headless/src/controllers/commerce/facets/core/numeric/headless-commerce-numeric-facet.ts

```diff
@@ -0,0 +1,56 @@
+import {CommerceEngine} from '../../../../../app/commerce-engine/commerce-engine';
+import {
+  toggleExcludeNumericFacetValue,
+  toggleSelectNumericFacetValue,
+} from '../../../../../features/facets/range-facets/numeric-facet-set/numeric-facet-actions';
+import {
+  CoreCommerceFacet,
+  CoreCommerceFacetOptions,
+  NumericFacetValue,
+  NumericRangeRequest,
+  buildCoreCommerceFacet,
+} from '../headless-core-commerce-facet';
+
+export type CommerceNumericFacetOptions = Omit<
+  CoreCommerceFacetOptions,
+  'toggleSelectActionCreator' | 'toggleExcludeActionCreator'
+>;
+
+/**
+ * The `CommerceNumericFacet` controller offers a high-level programming interface for implementing numeric commerce
+ * facet UI component.
+ */
+export interface CommerceNumericFacet
+  extends CoreCommerceFacet<NumericRangeRequest, NumericFacetValue> {}
```

Comment:
The controller interface is now just a type alias. We could simplify its definition like this:
```suggestion
export type CommerceNumericFacet = CoreCommerceFacet<NumericRangeRequest, NumericFacetValue>;
```